### PR TITLE
saltminion 3006.5 - Corrected url/64_py3 and checksum/64_py3

### PR DESCRIPTION
**Pull request by LeaDevelop**
Path to saltminion .exe on official saltstack repository has changed through time and chocolatey packages will result in failure to install. It makes sense to keep history version for 3006.x up to date as it is a LTS version.
https://docs.saltproject.io/salt/install-guide/en/latest/topics/salt-version-support-lifecycle.html#version-support-lifecycle

-  I created directory and pulled chocolateyuninstall.ps1 script, it is untouched.
-  chocolateyinstall.ps1 script got url corrections for both executables including the checksums
-  saltminion.nuspc got url corrections for documentation and pointed to up to date official logo

### CODE REVIEW

File: 3006-5/saltminion.nuspec [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <!-- == PACKAGE SPECIFIC SECTION == -->
+    <id>saltminion</id>
+    <version>3006.5.0</version>
+    <owners>Derek Murawsky, Alex Clay, espoelstra, ArthurREGNARD</owners>
+
+    <!-- == SOFTWARE SPECIFIC SECTION == -->
+    <title>Salt Minion (Install)</title>
+    <authors>SaltStack</authors>
```
Comment (twangboy) [inline 3006-5/saltminion.nuspec]: Salt Project

File: 3006-5/saltminion.nuspec [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    <projectUrl>https://www.saltstack.com/</projectUrl>
```
Comment (twangboy) [inline 3006-5/saltminion.nuspec]: www.saltproject.io

File: 3006-5/saltminion.nuspec [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    <iconUrl>https://saltproject.io/images/logo.svg</iconUrl>
+    <licenseUrl>https://github.com/saltstack/salt/blob/master/LICENSE</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <projectSourceUrl>https://github.com/saltstack/salt</projectSourceUrl>
+    <packageSourceUrl>https://github.com/barkingfoodog/ChocolateySaltMinion</packageSourceUrl>
+    <docsUrl>https://docs.saltstack.com/en/3006/contents.html</docsUrl>
```
Comment (twangboy) [inline 3006-5/saltminion.nuspec]: https://docs.saltproject.io/en/3006/contents.html

**Review by twangboy — CHANGES_REQUESTED** [review]

**Review by twangboy — APPROVED** [review]

**Comment by barkingfoodog** [comment]
Hi, thank you for the update! Would you please move the `tools/` directory one level higher and eliminate the `saltminion/` directory? This will allow the relative path in `saltminion.nuspec` to locate the files under `tools/`.

**Comment by LeaDevelop** [comment]
@barkingfoodog PR was aligned with master branch and suggested improvements done, ready for review

**Pull request merged**